edumanage/models.py

```diff
@@ -542,6 +543,8 @@ class InstServer(models.Model):
 
     secret = models.CharField(max_length=80)
     proto = models.CharField(max_length=12, choices=RADPROTOS, default=RADPROTOS.UDP)
+    psk_identity = models.CharField(max_length=64, null=True, blank=True, help_text=_("Network Access Identifier (user@realm)"))
```

Comment:
I see draft-ietf-radext-tls-psk specifically asks that:
> Implementations MUST support PSK Identities of 128 octets
.... perhaps this should be
```suggestion
    psk_identity = models.CharField(max_length=128, null=True, blank=True, help_text=_("Network Access Identifier (user@realm)"))
```
(and migration changed too) ?

Reply:
The length was based on the NAI implementation, but actually this is probably better.